shopify/finance-dashboard-app-private-remix-ssr/api/models/shopifyOrder/actions/update.ts

```diff
@@ -0,0 +1,72 @@
+import { applyParams, save, ActionOptions, logger } from "gadget-server";
+import { preventCrossShopDataAccess } from "gadget-server/shopify";
+import { isFullPage } from "@notionhq/client";
+import { enqueueNotionJob, getNotionOrder } from "../utils";
+import { JSONValue } from "@gadget-client/finance-dashboard-app";
+import { api } from "gadget-server";
+
+export const run: ActionRun = async ({ params, record }) => {
+  applyParams(params, record);
+  await preventCrossShopDataAccess(params, record);
+  await save(record);
+};
+
+const updateOrderInNotion = async (
```

Comment:
Just curious why this function is here and the create and fetch are in a utils file?